### PR TITLE
fix(live refresh): Tweak the text and tooltip conditionally.

### PR DESCRIPTION
**Pull request by Camwyn**
Change the text for when v1 is active. Add filters to change the text if Filter Bar is active and v1
is active. Tweak the tooltip text.

📷 V1 with Filter Bar: http://p.tri.be/L4Z8Ws
📷 V1 no FIlter Bar: http://p.tri.be/xzBjun
📷 V2 no Filter Bar: http://p.tri.be/PFdrGg (the shifted settings are a separate ticket)
📷 V2 with Filter Bar - on Filters tab: http://p.tri.be/yuwqwh

[FBAR-79]

[FBAR-79]: https://moderntribe.atlassian.net/browse/FBAR-79

### CODE REVIEW

File: src/admin-views/tribe-options-general.php [hunk 1]
```diff
@@ -265,11 +265,40 @@
 ];
 
 if ( tribe_events_views_v2_is_enabled() ) {
+	// Push the control to the Filters tab
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-general.php]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 24851163156e8e87501a342465930a61</sub>

File: src/admin-views/tribe-options-general.php [hunk 2]
```diff
@@ -265,11 +265,40 @@
...
+	/**
+	 * Filters the text for the "automatic" option.
+	 *
+	 * @since TBD
+	 *
+	 * @param string the displayed text.
+	 */
+	$automatic_text = apply_filters(
+		'tribe_events_liveupdate_automatic_label_text',
+		__( 'Enabled: datepicker selections automatically update calendar views.', 'the-events-calendar')
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-general.php]: * :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: a3a04875f5d1e7186dc5d51f507df07f</sub>

File: src/admin-views/tribe-options-general.php [hunk 3]
```diff
@@ -265,11 +265,40 @@
...
+	);
+	/**
+	 * Filters the text for the "manual" option.
+	 *
+	 * @since TBD
+	 *
+	 * @param string the displayed text.
+	 */
+	$manual_text    = apply_filters(
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-general.php]: :warning: Equals sign not aligned correctly; expected 1 space but found 4 spaces

<sub>hash: 66c37fe712345c0afebcbed074819a48</sub>

File: src/admin-views/tribe-options-general.php [hunk 4]
```diff
@@ -265,11 +265,40 @@
...
+		'tribe_events_liveupdate_manual_label_text',
+		__( 'Disabled: users must click Find Events to search by date.', 'the-events-calendar')
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-general.php]: * :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 43e20df4bb8620d1dacc9f314433e5ba</sub>

File: src/admin-views/tribe-options-general.php [hunk 5]
```diff
@@ -265,11 +265,40 @@
...
+	);
+
+	$filter_activation['liveFiltersUpdate']['options']['automatic'] = $automatic_text;
+	$filter_activation['liveFiltersUpdate']['options']['manual'] = $manual_text;
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-general.php]: :warning: Equals sign not aligned with surrounding assignments; expected 4 spaces but found 1 space

<sub>hash: 4205c0712d3cbe8ffa4ec41e1d0a4906</sub>

File: src/admin-views/tribe-options-general.php [hunk 6]
```diff
@@ -265,11 +265,40 @@
...
+	$filter_activation['liveFiltersUpdate']['label'] = esc_html__( 'Live Refresh', 'the-events-calendar' );
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-general.php]: :warning: Equals sign not aligned with surrounding assignments; expected 16 spaces but found 1 space

<sub>hash: df53064cdadc65c845a6b2615dc449c1</sub>

File: src/admin-views/tribe-options-general.php [hunk 7]
```diff
@@ -265,11 +265,40 @@
...
+
+	// Insert the control
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-general.php]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 676f03860de41800f0cbf5c1f69a5f4a</sub>

**Comment by lucatume** [comment]
I've restarted the build as I think the failure is a false negative due to timeout issues.

File: readme.txt [hunk 8]
```diff
@@ -224,9 +224,11 @@ Remember to always make a backup of your database and files before updating!
 
 * Tweak - Minify the Freemius svg assets. [TEC-3215]
 * Tweak - Remove "(beta)" label from URL source type of import. [TEC-3289]
+* Tweak - Reword the Filter Activation setting's labels for better clarity. Add filters `tribe_events_liveupdate_automatic_label_text` and `tribe_events_liveupdate_manual_label_text` to allow Filter Bar to include itself in those lables. [FBAR-79]
```
Comment (lucatume) [inline readme.txt]: Please, break this changelog entry in two as they would target 2 diff. audiences; one for the UI update, one for developers.

**Pull request merged**